mkdocs.yml

```diff
@@ -187,6 +187,7 @@ nav:
       - Debugging: eventing/debugging/index.md
       - Accessing CloudEvent traces: eventing/accessing-traces.md
       - Metrics API: eventing/metrics.md
+      - Experimental Features: eventing/experimental-features.md
```

Comment:
Should probably go in the admin guide if they all require configmap editing?

Reply:
:shrug: I see it's the same for serving...